create_test.go

```diff
@@ -42,6 +42,17 @@ const (
 
 var testStrPID = fmt.Sprintf("%d", testPID)
 
+func mockCPUSetContent(contents map[string]string) error {
+	fileMode := os.FileMode(0600)
```

Comment:
You could just use `testFileMode` (from `main_test.go`) here.

Reply:
yes, I also consider whether to use testFileMode or use `os.FileMode(0600)` here